.gitignore

```diff
@@ -9,3 +9,5 @@
 /venv/*
 _certs/
 .coverage
+mobile_harness_gateway_pb2.py
```

Comment:
Should these be only in tools/ directory ?

Reply:
@iamjazzar , can you take a look?

Reply:
I've temporarily removed all mobile harness references from this PR.